### PR TITLE
feat: Location validation & categorization

### PR DESCRIPTION
**Pull request by aguirrekarina**
# Pull Request

Location and categorization service

## Description

I created the location service using the Mapbox API to calculate the distance between the given location and the store, it was also used to verify that the location belongs within Bolivia, and then categorize it as agreed.
## Link To Tasks

[Location validaton and categorization](https://app.clickup.com/t/8689mr819)
## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality
      to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by JorgeHB69** [comment]
Task linked: [CU-8689mr819 Location validation & categorization](https://app.clickup.com/t/8689mr819)

File: DistributionCenter.sln.DotSettings.user
Comment (LuisHerbas98) [inline DistributionCenter.sln.DotSettings.user:1]: Is this file really needed in the repository?
I think it should be removed and added to the `.gitignore`.

**Review by MorveN11 — APPROVED** [review]
Good Job!

**Review by JeferssonCL — APPROVED** [review]
